Userland/Applications/TextEditor/main.cpp

```diff
@@ -88,13 +88,60 @@ int main(int argc, char** argv)
     text_widget.initialize_menubar(menubar);
     app->set_menubar(menubar);
 
-    if (file_to_edit)
-        text_widget.open_file(file_to_edit);
-    else
+    // If a colon exists in the file name argument, the user may have specified line and column numbers.
+    // First check if the file exists before doing any further processing. If so, open it.
+    // Otherwise, split apart the line and column numbers from the file name, open the file, and jump to the specified line/column.
+
+    if (file_to_edit) {
+        if (StringView(file_to_edit).contains(':')) {
+            if (Core::File::exists(file_to_edit)) {
+                // A file exists with the full specified name, open it without doing any further processing.
+                text_widget.open_file(file_to_edit);
+            } else {
+                // A file doesn't exist with the full specified name, maybe the user entered line/column coordinates?
+                Regex<PosixExtended> line_and_column_number("^(.+?)(:([0-9]+):?([0-9]+)?)?$");
+                RegexResult result = match(file_to_edit, line_and_column_number, PosixFlags::Global | PosixFlags::Multiline | PosixFlags::Ungreedy);
+
+                // Match 0 group 0: file name
+                // Match 0 group 1: line number
+                // Match 0 group 2: column number
+
+                if (result.capture_group_matches.at(0).size() > 3) {
```

Comment:
Just a nitpick,
Maybe extract this `result.capture_group_matches.at(0)` above just once? `.at(0).at(0)` looks rather odd

```c++
auto& groups = result.capture_group_matches.at(0);
if (groups.size() > 3)
    ...
```

Reply:
Right, thanks!